calyx-ir/src/control.rs

```diff
@@ -175,8 +263,20 @@ pub enum Control {
     Enable(Enable),
     /// Control statement that does nothing.
     Empty(Empty),
-    /// Runs the control for a static group.
-    StaticEnable(StaticEnable),
+    /// Static Control
+    Static(StaticControl),
+}
+
+/// Control AST nodes.
+#[derive(Debug)]
+pub enum StaticControl {
+    /// Essentially a Static While Loop
+    Repeat(StaticRepeat),
+    Enable(StaticEnable),
+    Par(StaticPar),
+    Seq(StaticSeq),
+    If(StaticIf),
+    Empty(Empty),
```

Comment:
What is the use case for a Static Empty? Does it show up if we have `if cond { foo }` without the `else` branch?

Reply:
yeah